lib/PuppeteerSharp/PuppeteerSharp.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;netcoreapp2.1;netstandard2.0</TargetFrameworks>

```

Comment:
As I mentioned PuppeteerSharp should only be a netstandard project

Reply:
If remove it from `PuppeteerSharp.csproj` and `PuppeteerSharp.TestServer.csproj` then it's not possible to build the dll:
```
dotnet build -f netcoreapp2.1
```
How to do it without a definition inside csproj?

Reply:
I think you should be able to use msbuild on Linux https://ci.appveyor.com/project/kblok/puppeteer-sharp/build/1.0.1626/job/tlxntjkyfpxd465y#L121

Reply:
With mono msbuild all projects compile successfully except `PuppeteerSharp.Tests.csproj`. What could be wrong there? https://travis-ci.org/HarpyWar/puppeteer-sharp/jobs/433140784#L2276-L2277

Also should mono be included into travis to build the solution, or it's enough in appveyour?

Reply:
AppVeyor now supports Linux, so We would build there.
But your work here would help us to setup AppVeyor, in fact, we could close this task setting up AppVeyor.
So:
1. I think you only need to `dotnet test -f netcoreapp2.0 -s test.runsettings` You don't need anything else.
2. We need to be able to create a self-signed certificate as we do in Windows https://github.com/kblok/puppeteer-sharp/blob/master/appveyor.yml#L22
3. You would need to incorporate this PR https://github.com/kblok/puppeteer-sharp/pull/430. It will do a chmod on the downloaded file.

We have our channel (Puppeteer-Sharp) on the Puppeteer's slack workspace https://puppeteer.slack.com/messages fill free to jump there to chat.